spec/amp-cache-debugging.md

```diff
@@ -0,0 +1,23 @@
+# AMP Cache Debugging
+
+**Why is my doc broken on an AMP cache?**
+
+Valid AMP documents typically appear and behave the same on AMP Caches as they
+do on the origin. However, there are some components and server configurations
+that can be problematic.
+
+If a particular document appears and behaves as expected on your origin, but not
+when viewed via the cache (how to map origin URLs to [Google's AMP Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)),
+first open the developer console in your browser, and ensure no errors or
```

Comment:
Thanks, I was trying to be browser neutral but ended up being confusing--I've adjusted the language a bit in 5765a50f. Should I also specifically call out Chrome?